src/datachain/lib/dc.py

```diff
@@ -246,6 +247,8 @@ def __init__(self, *args, settings: Optional[dict] = None, **kwargs):
             **kwargs,
             indexing_column_types=File._datachain_column_types,
         )
+        send_telemetry_once("datachain_init", **kwargs)
```

Comment:
can we make it part of the telemetry package please? to keep code here simpler 

I mean the "once" part implementation

Reply:
https://github.com/iterative/telemetry-python/pull/59 PTAL cc. @shcheklein